File: app/src/main/java/pl/szczodrzynski/edziennik/ui/modules/agenda/lessonchange/LessonChangeEventRenderer.java
package pl.szczodrzynski.edziennik.ui.modules.agenda.lessonchange;

import android.view.View;
import android.widget.TextView;

import androidx.cardview.widget.CardView;

import com.github.tibolte.agendacalendarview.render.EventRenderer;

import pl.szczodrzynski.edziennik.R;

public class LessonChangeEventRenderer extends EventRenderer<LessonChangeEvent> {
    @Override
    public void render(View view, LessonChangeEvent event) {
        CardView card = view.findViewById(R.id.lesson_change_card);
        TextView changeText = view.findViewById(R.id.lesson_change_text);
        TextView changeCount = view.findViewById(R.id.lesson_change_count);
        card.setCardBackgroundColor(event.getColor());
        changeText.setTextColor(event.getTextColor());
        changeCount.setTextColor(event.getTextColor());
        changeCount.setText(String.valueOf(event.getLessonChangeCount()));
    }

    @Override
    public int getEventLayout() {
        return R.layout.agenda_event_lesson_change;
    }
}

File: app/src/main/java/pl/szczodrzynski/edziennik/ui/modules/agenda/lessonchange/LessonChangeEventRenderer.kt
package pl.szczodrzynski.edziennik.ui.modules.agenda.lessonchange

import android.view.View
import android.widget.TextView
import androidx.cardview.widget.CardView
import com.github.tibolte.agendacalendarview.render.EventRenderer
import pl.szczodrzynski.edziennik.R

class LessonChangeEventRenderer : EventRenderer<LessonChangeEvent>() {
    override fun render(view: View?, event: LessonChangeEvent) {
        val card = view?.findViewById<CardView>(R.id.lesson_change_card)
        val changeText = view?.findViewById<TextView>(R.id.lesson_change_text)
        val changeCount = view?.findViewById<TextView>(R.id.lessonChangeCount)
        card?.setCardBackgroundColor(event.color)
        changeText?.setTextColor(event.textColor)
        changeCount?.setTextColor(event.textColor)
        changeCount?.text = event.lessonChangeCount.toString()
    }

    override fun getEventLayout(): Int = R.layout.agenda_event_lesson_change
}
